### DISCUSSION TITLE
Hooks

### DISCUSSION THREAD

**Discussion by encodetocode** [CodeDredd/laravel-soap#216] [General]
Dears,

Well done for your amazing package,

I would like to know if there are hooks in this package, that do hook before or after SOAP request?

Thanks,

**Comment by CodeDredd** [comment]
Hey,

sorry for the late answer. Was on vacation 🍹 

Well depends what version you mean and what you under hook understand.
Since version v3.beta.2 there are EVENTS see -> https://github.com/CodeDredd/laravel-soap/releases/tag/v3.0.0.beta.2
